crates/ra_project_model/src/sysroot.rs

```diff
@@ -126,7 +126,6 @@ core
 alloc
 collections
 libc
-panic_unwind
```

Comment:
Why was this removed?

Reply:
There's already one bellow (hence the dedup call in fst construction code, just to be safe :D)